tests/tests_cli/test_download_files.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of cernopendata-client.
+#
+# Copyright (C) 2020 CERN.
+#
+# cernopendata-client is free software; you can redistribute it and/or modify
+# it under the terms of the GPLv3 license; see LICENSE file for more details.
+
+"""cernopendata-client cli command download-files test."""
+
+from click.testing import CliRunner
+from cernopendata_client.cli import download_files
+
+
+def test_download_files():
+    """Test download_files() command."""
+    test_download_files = CliRunner()
+    test_result = test_download_files.invoke(download_files, ["--recid", 3005])
+    assert test_result.exit_code == 0
+    assert test_result.output.endswith("\nDownload completed!\n")
```

Comment:
You can also test whether `3005/0d0714743f0204ed3c0144941e6ce248.configFile.py` file is well present and whether its size is 3644 bytes.

BTW it may be necessary before running this test case to delete that file if it is present... so that we don't get false positives...